src/olympia/blocklist/admin.py

```diff
@@ -25,46 +24,42 @@
 GUID_FULL_LOAD_LIMIT = 100
 
 
+def _get_version_choices(obj, field):
+    default = obj._meta.get_field(field).default
+    choices = [
+        (version, version) for version in (
+            [default] + list(obj.addon_versions.keys())
+        )
+    ]
+    value = getattr(obj, field)
+    if value and (value, value) not in choices:
+        choices = [(getattr(obj, field), '(invalid)')] + choices
```

Comment:
I'm not sure I fully understood what this is doing. How can the `(vaue, value) not in choices` ever be `True` here? Where is `(invalid)` coming from again, I couldn't find it.

Reply:
... yeah, it's become a little detached from the original implementation that was on the back of a pull request discussion.

The call would be something like `_get_version_choices(block, 'min_version')` so `value` would be a version string.  `(value, value)` wouldn't be in `choices` if the version string (`Block.min_version`) wasn't one of the versions of the addon the block was referencing.  This could either be because the block was created as part of a multiple block submission and the admin creating it chose the broadest version range that would block everything, or because it was a new block and the `min_version` was passed as a `GET` param.
(it's also used for `max_version` too)

Reply:
https://github.com/mozilla/addons-server/pull/13342/commits/8bfabf161520ec4f1aedabcec1a3cd4e3eee0baf